huaweicloud/services/waf/resource_huaweicloud_waf_domain.go

```diff
@@ -14,6 +14,8 @@ import (
 	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils/logp"
 )
 
+var PaidType = "prePaid"
```

Comment:
it's better to define it as **[]string**

Reply:
var PaidType indicates the default value, so I think string is better